release notes/v0.41.0.md

```diff
@@ -0,0 +1,64 @@
+k6 v0.41.0 is here! :tada: It has relatively few user-facing changes, but includes massive internal improvements that will pave the way [for some awesome features](#roadmap-and-future-plans) in the near future. Unfortunately, we also had to make a few [minor breaking changes](#breaking-changes) to allow us to do that.
+
+## Breaking changes
+
+### Changes in the `url`, `iter` and `vu` system metric tags
+
+As we warned in both the k6 [v0.39.0](https://github.com/grafana/k6/blob/master/release%20notes/v0.39.0.md#refactoring-metrics) and [v0.40.0](https://github.com/grafana/k6/blob/master/release%20notes/v0.40.0.md#refactoring-metrics) release notes, we have been fundamentally [refactoring the metrics sub-systems](#metric-time-series) of k6. We now have efficient support for time series, and that required a few minor user-facing breaking changes:
+
+1. If [URL grouping](https://k6.io/docs/using-k6/http-requests/#url-grouping) is used for HTTP requests (that is, if the `name` tag is somehow specified), then the `url` tag in the resulting `http_req_*` metric samples will also have the same value as the `name` tag. Previously this was done only for [the `cloud` output](https://k6.io/docs/results-output/real-time/cloud/), but now it will be done universally ([#2703](https://github.com/grafana/k6/pull/2703)).
+2. The [`vu` and `iter` system tags](https://k6.io/docs/using-k6/tags-and-groups/#system-tags), which are disabled by default, have been transformed into [high-cardinality metrics metadata](#internal-support-for-high-cardinality-metrics-metadata) instead. It means that they will no longer be usable in thresholds, and various outputs may emit them differently or ignore them completely ([#2726](https://github.com/grafana/k6/pull/2726)).
+
+### Changes in the Go metrics APIs
+
+While the user-facing changes from our metrics refactoring are few and relatively minor, and there are no changes to JavaScript APIs yet, we have extensively refactored our internal Go APIs ([#2594](https://github.com/grafana/k6/pull/2594), [#2726](https://github.com/grafana/k6/pull/2726), [#2727](https://github.com/grafana/k6/pull/2727)). The [`metrics.Sample`](https://github.com/grafana/k6/blob/8a74171c8b43f48fb53078e0bcfa7851828124bc/metrics/sample.go#L11-L33) and [`metrics.TagSet`](https://github.com/grafana/k6/blob/8a74171c8b43f48fb53078e0bcfa7851828124bc/metrics/tags.go#L16-L157) types are now entirely different. We also have high-cardinality metadata attributes in each sample and at the VU level (see [here](https://github.com/grafana/k6/blob/8a74171c8b43f48fb53078e0bcfa7851828124bc/metrics/tags.go#L159-L231) and [here](https://github.com/grafana/k6/blob/8a74171c8b43f48fb53078e0bcfa7851828124bc/lib/vu_state.go#L83-L119)).
```

Comment:
"Here" as link text doesn't say anything about what it takes the reader to, and it's inaccessible for screen readers / crawlers. If I understand, this parenthetical also refers to two different pieces.

I'd split it. Something like this:

> We also have high-cardinality metadata attributes in [each sample](https://github.com/grafana/k6/blob/8a74171c8b43f48fb53078e0bcfa7851828124bc/metrics/tags.go#L159-L231) and at the [VU level](https://github.com/grafana/k6/blob/8a74171c8b43f48fb53078e0bcfa7851828124bc/lib/vu_state.go#L83-L119).

I'm not putting that edit in myself though, because I'm not sure. You could also write the link as "code", to give a hint about where the reader would go.

> ...in each sample ([code](code)) and VU level ([code]).

Reply:
Resolved with the [latest force-push](https://github.com/grafana/k6/compare/0b6ec90828afac81746a6d300c878006c57b5ac7..f68483e91bdb7c70579acea6ada26a549c7a6e6e)